src/nova_security_handler.erl

```diff
@@ -35,7 +36,11 @@ execute(Req = #{host := Host}, Env = #{secure := {Module, Function}}) ->
             {ok, Req0, _Env0} = nova_router:render_status_page(Host, 401, #{}, Req, Env),
             Req1 = cowboy_req:reply(401, Req0),
             {stop, Req1}
-    catch _Class:_Reason ->
+    catch Class:Reason:Stacktrace ->
```

Comment:
Maybe we should make a macro of this so iff DEV is set (or similar we fetch stacktrace otherwise not). Think we have this in some additional places as well but a very simple DoS-attack would be to continuously query the API with bad credentials. 

Think we need to address this soon since it's a fairly easy attack :(

Reply:
But if you have bad credentials you should handle it in your application. Right now if you get a crash in the security handler we don't get anything back at all and it is tough to debug why it goes wrong. And using dbg to get a stacktrace or understand what went wrong is not a nice experience for development.